examples/multiomics_mogonet/configs/MOGONET_ROSMAP.yaml

```diff
@@ -0,0 +1,13 @@
+DATASET:
+  NAME: "ROSMAP"
+  URL: "https://github.com/pykale/data/raw/main/multiomics/ROSMAP.zip"
+  NUM_MODALITIES: 3
+  NUM_CLASSES: 2
+
+SOLVER:
+  MAX_EPOCHS_PRETRAIN: 5 # For quick testing
+  MAX_EPOCHS: 10  # For quick testing
```

Comment:
Should the file name reflect this?

Reply:
I have renamed the file to "MOGONET_ROSMAP_quick_test.yaml".